EdgeOfEmpireBot/EdgeOfEmpireBot/Service/MessageService.cs

```diff
@@ -89,7 +103,25 @@ private async Task ProcessCommand(string command, ISocketMessageChannel channel)
                     }
                 case "add":
                     {
-                        AddCommand(command,channel);
+                        var msg =this.dataService.AddCommand(command);
+                        await SendMessage(msg, channel);
+                        break;
+                    }
+                case "gameRequest":
```

Comment:
Could probably convert the words into upper case or lowercase so that way you don't need to worry about capital letters